regression/find-repos.ts

```diff
@@ -55,9 +57,18 @@ async function getReposFromGitHub(): Promise<GHRepo[]> {
       if (process.env.GITHUB_API_KEY) {
         options.headers = { Authorization: `token ${process.env.GITHUB_API_KEY}` };
       }
+      let proxyAgent;
+      const httpsProxy = process.env.HTTPS_PROXY;
+      if (httpsProxy) {
+        // https://github.com/axios/axios/issues/3459
+        proxyAgent = new (HttpsProxyAgent as any)(httpsProxy);
+      }
       const res = await axios.get(
         `https://api.github.com/orgs/HL7/repos?sort=full_name&per_page=100&page=${page}`,
-        options
+        {
+          httpAgent: proxyAgent,
+          headers: options
+        }
```

Comment:
I think this should use `axiosGet` too.  I suspect it doesn't use it now because it needs to pass in that extra option (the authorization header).  Maybe instead of the 2nd argument to `axiosGet` being `responseType?: any`, it should be `options?: any`.  Then where you need the response type, you pass in `{ responseType: 'arraybuffer' }` and here, if you need the authorization header, you pass in ``{ headers: { Authorization: `token ${process.env.GITHUB_API_KEY}` } }``.  Then that would be more flexible if we ever have to pass in other options as well.

Reply:
Pushed these changes, thank you.